actionview/lib/action_view/test_case.rb

```diff
@@ -108,6 +189,26 @@ def include_helper_modules!
         end
       end
 
+      included do
+        class_attribute :content_class, instance_accessor: false, default: Class.new(Content)
```

Comment:
@rafaelfranca is this class a `Content` descendant for troubleshooting and naming purposes? Could it be `default: Class.new(SimpleDelegator)` directly, cutting out the `Content` class entirely?

Reply:
It doesn't need to be a descendant. I just changed that. But I prefer to have an name for debug purpose, and documentation so people can search. I just used the Ruby 3.3 set_temporary_name if it is present.